Makefile

```diff
@@ -18,6 +18,7 @@ prod: prod-image
 
 prod-test: prod-image
 	. environment/env-prod; docker run \
+		-t \
```

Comment:
What is the `-t` used for in this case?

Reply:
It allocates a pseudo-TTY so that we can send `SIGINT` to the container running the tests and stop early.